### PR TITLE
Changes to make building a request easier

### PR DESCRIPTION
**Pull request by hyperioxx**
#134

### CODE REVIEW

**Comment by peterdonald3** [comment]
Looks okay apart from the fmt. statement

**Comment by hyperioxx** [comment]
> Looks okay apart from the fmt. statement


@peterdonald3 Yeah missed that, now been removed !

File: kalbi.go [hunk 1]
```diff
@@ -132,6 +147,7 @@ func (ed *SipStack) Start() {
 		msg := <-ed.TransportChannel
 		event := ed.TransManager.Handle(msg)
 		message := event.GetSipMessage()
+		fmt.Print(message)
```
Comment (DeWarner) [inline kalbi.go:150]:
```suggestion
		log.Log.Info(message)
```

**Review by DeWarner — CHANGES_REQUESTED** [review]

File: kalbi.go [hunk 2]
```diff
@@ -1,7 +1,8 @@
 package kalbi
 
 import (
-	"github.com/KalbiProject/kalbi/interfaces"
+	"fmt"
+
```
Comment (DeWarner) [inline kalbi.go:5]:
```suggestion
```

File: kalbi.go [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
 	"github.com/KalbiProject/kalbi/log"
```
Comment (DeWarner) [inline kalbi.go:6]:
```suggestion
	"github.com/KalbiProject/kalbi/log"
```

File: kalbi.go (hunk 1 above)
Comment (DeWarner) [inline kalbi.go:150]:
```suggestion
		log.Log.Info(message)
```

**Review by DeWarner — DISMISSED** [review]

File: examples/basicUAC/main.go [hunk 4]
```diff
@@ -98,7 +92,7 @@ func (c *Client) ReRegister(event interfaces.SipEventObject) {
 
 // HandleResponses
 // example Client struct method to handle responses
-func (c *Client) HandleResponses(event interfaces.SipEventObject) {
+func (c *Client) HandleResponses(event message.SipEventObject) {
 
 	response := event.GetTransaction()
 	fmt.Println(string(event.GetSipMessage().Src))
```
Comment (peterdonald3) [inline examples/basicUAC/main.go:98]: to delete

Comment (peterdonald3) [reply examples/basicUAC/main.go:98]: \ and import

**Review by peterdonald3 — APPROVED** [review]
Approved, but needs a new issue raising for rogue fmt.Println()s

**Review by DeWarner — APPROVED** [review]

**Pull request merged**